nest-forms-backend/src/nases/nases.service.ts

```diff
@@ -429,14 +429,29 @@ export default class NasesService {
     }
 
     // Send to nases
-    await this.nasesConsumerService.sendToNasesAndUpdateState(
+    const isSent = await this.nasesConsumerService.sendToNasesAndUpdateState(
       jwt,
       form,
       data,
       formDefinition,
       user.sub,
     )
 
+    if (!isSent) {
+      // TODO: It would be better to rewrite how sendToNasesAndUpdateState works or use a different function
+      // TODO: This also breaks if anything throws between setting to SENDING_TO_NASES and reverting to DRAFT
+      // today, if we don't revert the state in this case, the form can't be sent again
+      await this.formsService.updateForm(data.formId, {
```

Comment:
Is this a supported thing? draft + error? I won't notice this in metabese if looking for errors

Reply:
not supported by frontend - in the sense frontend uses the error field only when form state is error as well. But "supported" in the sense that it does not break anything as far as I could tell. So kind-of misused here for admin purposes, so that we may notice the forms where this happens and they don't get lost among the other drafts.

On BE, the error is mostly used for us to know what state is the form in - from what I could find, as far as sending the form, [this](https://github.com/bratislava/konto.bratislava.sk/blob/558f9cb4b3abd5cd427102d929895b8f1e82227c/nest-forms-backend/src/forms/forms.helper.ts#L100) is the only place where it comes into play and that's again only if form state is ERROR as well